spec/services/random_phrase_spec.rb

```diff
@@ -21,5 +21,13 @@
 
       expect(phrase.to_s.length).to eq 24 # 20 chars + 4 spaces
     end
+
+    it 'does not contain the letters I L O U' do
+      100.times do
```

Comment:
would it be possible to do this fewer than 100 times and still have an effective test? I imagine 100x is kinda slow but maybe I am wrong